velox/functions/lib/DateTime.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "DateTimeFormatter.h"
+
+namespace facebook::velox {
+struct Timestamp;
+}
+
+namespace facebook::velox::functions {
+class DateTime {
```

Comment:
It seems that class is using some formatters, while it's name is "DateTime", which I understand as it representing a DateTime type.
Should we name it differently?
And add a comment why do we need this class.
We have too many classes w/o any hint what are they for.

Reply:
We still have this class named "DateTime", while it does not really represent that.
Let's come up with a different name?

Also, why couldn't we have DateTimeFormatter to have "parse" and  "format"?
This class seems redundant and my inability to propose a proper name for it (it should be DateTimeFormatter, really, but we have that already) confirms it.

Reply:
I am actually thinking about the same thing. My previous thought was DateTime can have functionality more than just parse and format, a more generic entity that deals anything with a time standard. But so far I have not seen use cases other than parse and format. Maybe we should just let MySQLDateTimeFormatter extend DateTimeFormatter and remove this class for now until this class is really needed. I'll do the refactor.